### PR TITLE
[Designs] Update designs schema to `v1beta1`

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
// @aabidsofi19 @leecalcote

File: schemas/constructs/v1beta1/designs.json [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  "description": "Designs are your primary tool for collaborative authorship of your infrastructure, workflow, and processes.",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Name of the design; a descriptive, but concise title for the design document."
+    },
+    "schemaVersion": {
+      "$ref": "../v1alpha2/core.json#/definitions/versionString",
+      "description": "Specifies the version of the schema to which the design  conforms."
```
Comment (leecalcote) [inline schemas/constructs/v1beta1/designs.json:13]:
```suggestion
      "description": "Specifies the version of the schema to which the design conforms."

```

File: schemas/constructs/v1beta1/designs.json [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    },
+    "version": {
+      "$ref": "../v1alpha2/core.json#/definitions/semverString",
+      "description": "Revision of the design as expressed by an auto-incremented, SemVer-compliant version number. May be manually set by a user or third-party system, but will always be required to be of version number higher than the previously defined version number.",
+      "minLength": 1,
+      "maxLength": 50
+    },
+    "components": {
+      "description": "Map of component IDs to their corresponding component declarations",
```
Comment (leecalcote) [inline schemas/constructs/v1beta1/designs.json:22]:
```suggestion
      "description": "Map of component instances to their corresponding component declarations",

```

File: schemas/constructs/v1beta1/designs.json [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+          "description": "Component and it's configuration.",
+          "properties": {
+            "version": {
+              "$ref": "./component.json#/properties/component/properties/version"
+            },
+            "kind": {
+              "$ref": "./component.json#/properties/component/properties/kind"
+            },
+            "configuration": {
+              "$comment": "The configuration of the component. The configuration is based on the schema defined inside the component definition.",
```
Comment (leecalcote) [inline schemas/constructs/v1beta1/designs.json:55]:
```suggestion
              "$comment": "The configuration of the component. The configuration is based on the schema defined within the component definition.",

```

File: schemas/constructs/v1beta1/designs.json [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        "version": {
+          "$ref": "./component.json#/properties/version"
+        },
+        "name": {
+          "$ref": "../v1alpha2/core.json#/definitions/inputString"
+        },
+        "model": {
+          "$ref": "./component.json#/properties/model"
+        },
+        "metadata": {
```
Comment (leecalcote) [inline schemas/constructs/v1beta1/designs.json:41]: It'd be nice to move metadata above model and not in-between model and component.

File: schemas/constructs/v1beta1/designs.json [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "layers"
+      ]
+    },
+    "relationships": {
```
Comment (leecalcote) [inline schemas/constructs/v1beta1/designs.json:94]: Why is this not a $ref?

**Comment by leecalcote** [comment]
Was the other PR closed?

**Pull request merged**